source/_includes/common-tasks/enable_i2c.md

```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System which is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to use I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Configure the Raspberry Hats binary sensor integration to enable the I2C integration
```

Comment:
This is only required if you use the I2C hat. I2C can also be used without that hat, so I would just drop this line

```suggestion
```